### ISSUE TITLE
Stuck at "Your circles account is loading"

### ISSUE THREAD

**Issue by ligi** [CirclesUBI/circles-myxogastria#121]
I am stuck in this screen - it spins over night:

![Selection_247](https://user-images.githubusercontent.com/111600/96363958-05517f80-1138-11eb-9189-b0b4bcebd410.png)

tried with 2 browsers (same mnemonic) - firefox and brave

console log in FF:

```
XHRPOSThttps://o334096.ingest.sentry.io/api/5446133/envelope/?sentry_key=1c5b1a302d3d4e7a8037e67b31fefbd0&sentry_version=7
[HTTP/1.1 429 Too Many Requests 827ms]

[Error] Tried too many times waiting for conditi.. debug.js:24:10
CoreError: Tried too many times waiting for condition
    r index.js:191
    a index.js:79975
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    promise callback*B index.js:77781
    a index.js:77791
    D index.js:77796
    D index.js:77788
    n helpers.js:31
    setInterval handler*he</e.prototype._wrapTimeFunction/< trycatch.js:66
    e index.js:79950
    e index.js:79948
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    D index.js:77796
    D index.js:77788
    se index.js:79999
    ae index.js:79940
    t index.js:89288
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    promise callback*B index.js:77781
    a index.js:77791
    D index.js:77796
    D index.js:77788
    executeSafeTx index.js:89345
    t index.js:78935
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    promise callback*B index.js:77781
    a index.js:77791
    D index.js:77796
    D index.js:77788
    deploy index.js:78950
    e core.js:17
    l runtime.js:63
    _invoke runtime.js:293
    E runtime.js:118
    Babel 4
    p app-8dc5.js:1
    e core.js:152
    l runtime.js:63
    _invoke runtime.js:293
    E runtime.js:118
    Babel 4
    deploy core.js:151
    e actions.js:33
    l runtime.js:63
    _invoke runtime.js:293
    E runtime.js:118
    Babel 3
instrument.js:109:45
Symbol(TOO_MANY_ATTEMPTS) instrument.js:109:45
Uncaught (in promise) CoreError: Tried too many times waiting for condition
    r index.js:191
    a index.js:79975
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    promise callback*B index.js:77781
    a index.js:77791
    D index.js:77796
    D index.js:77788
    n helpers.js:31
    setInterval handler*he</e.prototype._wrapTimeFunction/< trycatch.js:66
    e index.js:79950
    e index.js:79948
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    D index.js:77796
    D index.js:77788
    se index.js:79999
    ae index.js:79940
    t index.js:89288
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    promise callback*B index.js:77781
    a index.js:77791
    D index.js:77796
    D index.js:77788
    executeSafeTx index.js:89345
    t index.js:78935
    l index.js:77320
    _invoke index.js:77461
    E index.js:77353
    B index.js:77772
    a index.js:77791
    promise callback*B index.js:77781
    a index.js:77791
    D index.js:77796
    D index.js:77788
    deploy index.js:78950
    e core.js:17
    l runtime.js:63
    _invoke runtime.js:293
    E runtime.js:118
    Babel 4
    p app-8dc5.js:1
    e core.js:152
    l runtime.js:63
    _invoke runtime.js:293
    E runtime.js:118
    Babel 4
    deploy core.js:151
    e actions.js:33
    l runtime.js:63
    _invoke runtime.js:293
    E runtime.js:118
    Babel 3
index.js:133:24

```

console log in brave:

https://gist.github.com/ligi/df2f3b4db0fd0d8d8bdb62cda760b511

**Comment by shayanb** [comment]
Same issue, with different error message. Already tried in 2 different browsers for the past 24hours, not fixed yet.

```
instrument.js:136 POST https://o334096.ingest.sentry.io/api/5446133/envelope/?sentry_key=1c5b1a302d3d4e7a8037e67b31fefbd0&sentry_version=7 429 (Too Many Requests)
(anonymous) @ instrument.js:136
(anonymous) @ fetch.js:43
e @ syncpromise.js:78
t.sendEvent @ fetch.js:41
e.sendEvent @ basebackend.js:33
e._sendEvent @ baseclient.js:321
t._sendEvent @ client.js:64
(anonymous) @ baseclient.js:361
onfulfilled @ syncpromise.js:137
(anonymous) @ syncpromise.js:66
_executeHandlers @ syncpromise.js:59
_attachHandler @ syncpromise.js:50
(anonymous) @ syncpromise.js:127
e @ syncpromise.js:78
e.then @ syncpromise.js:126
(anonymous) @ baseclient.js:352
e @ syncpromise.js:78
e._processEvent @ baseclient.js:350
e.captureEvent @ baseclient.js:95
e._invokeClient @ hub.js:316
e.captureEvent @ hub.js:168
t.finish @ transaction.js:100
t.finish @ idletransaction.js:128
(anonymous) @ idletransaction.js:148
n @ helpers.js:72
setTimeout (async)
(anonymous) @ trycatch.js:86
t.initSpanRecorder @ idletransaction.js:146
p @ hubextensions.js:87
g @ hubextensions.js:169
e._createRouteTransaction @ browsertracing.js:78
(anonymous) @ browsertracing.js:53
routingInstrumentation @ router.js:16
e.setupOnce @ browsertracing.js:53
(anonymous) @ integration.js:49
(anonymous) @ integration.js:63
S @ integration.js:61
e.setupIntegrations @ baseclient.js:147
e.bindClient @ hub.js:57
(anonymous) @ sdk.js:16
ve @ sdk.js:84
u @ sentry.js:13
885 @ index.js:14
c @ bootstrap:79
n @ bootstrap:45
(anonymous) @ bootstrap:152
(anonymous) @ app-8dc5.js:1
validation:1 Access to fetch at 'https://relay.circles.garden/api/v2/safes/0xDa1523dB637fA18F9c03b854A34E31A7BE9D1Cd8/funded/' from origin 'https://circles.garden' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
index.js:88688 PUT https://relay.circles.garden/api/v2/safes/0xDa1523dB637fA18F9c03b854A34E31A7BE9D1Cd8/funded/ net::ERR_FAILED
(anonymous) @ index.js:88688
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
$o @ index.js:88642
(anonymous) @ index.js:88745
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
Qo @ index.js:88718
(anonymous) @ index.js:89054
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
(anonymous) @ index.js:89063
(anonymous) @ index.js:78292
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
(anonymous) @ index.js:78307
(anonymous) @ core.js:17
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
p @ app-8dc5.js:1
(anonymous) @ core.js:48
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ core.js:47
(anonymous) @ actions.js:241
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ actions.js:229
(anonymous) @ index.js:8
dispatch @ redux.js:638
(anonymous) @ actions.js:141
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ actions.js:133
(anonymous) @ index.js:8
(anonymous) @ ValidationStatus.js:58
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ ValidationStatus.js:57
s @ react-dom.production.min.js:14
h @ react-dom.production.min.js:14
(anonymous) @ react-dom.production.min.js:14
g @ react-dom.production.min.js:15
st @ react-dom.production.min.js:52
ot @ react-dom.production.min.js:51
ut @ react-dom.production.min.js:52
pt @ react-dom.production.min.js:56
D @ react-dom.production.min.js:287
z @ react-dom.production.min.js:19
Qt @ react-dom.production.min.js:70
Jt @ react-dom.production.min.js:69
t.unstable_runWithPriority @ scheduler.production.min.js:19
Gi @ react-dom.production.min.js:122
P @ react-dom.production.min.js:287
Zt @ react-dom.production.min.js:68
n @ helpers.js:72
Show 71 more frames
asyncToGenerator.js:6 Uncaught (in promise) TypeError: Failed to fetch
Promise.then (async)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ ValidationStatus.js:57
s @ react-dom.production.min.js:14
h @ react-dom.production.min.js:14
(anonymous) @ react-dom.production.min.js:14
g @ react-dom.production.min.js:15
st @ react-dom.production.min.js:52
ot @ react-dom.production.min.js:51
ut @ react-dom.production.min.js:52
pt @ react-dom.production.min.js:56
D @ react-dom.production.min.js:287
z @ react-dom.production.min.js:19
Qt @ react-dom.production.min.js:70
Jt @ react-dom.production.min.js:69
t.unstable_runWithPriority @ scheduler.production.min.js:19
Gi @ react-dom.production.min.js:122
P @ react-dom.production.min.js:287
Zt @ react-dom.production.min.js:68
n @ helpers.js:72
validation:1 Access to fetch at 'https://relay.circles.garden/api/v2/safes/0xDa1523dB637fA18F9c03b854A34E31A7BE9D1Cd8/funded/' from origin 'https://circles.garden' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
index.js:88688 PUT https://relay.circles.garden/api/v2/safes/0xDa1523dB637fA18F9c03b854A34E31A7BE9D1Cd8/funded/ net::ERR_FAILED
(anonymous) @ index.js:88688
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
$o @ index.js:88642
(anonymous) @ index.js:88745
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
Qo @ index.js:88718
(anonymous) @ index.js:89054
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
(anonymous) @ index.js:89063
(anonymous) @ index.js:78292
l @ index.js:77320
(anonymous) @ index.js:77461
(anonymous) @ index.js:77353
B @ index.js:77772
a @ index.js:77791
(anonymous) @ index.js:77796
(anonymous) @ index.js:77788
(anonymous) @ index.js:78307
(anonymous) @ core.js:17
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
p @ app-8dc5.js:1
(anonymous) @ core.js:48
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ core.js:47
(anonymous) @ actions.js:241
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ actions.js:229
(anonymous) @ index.js:8
dispatch @ redux.js:638
(anonymous) @ actions.js:141
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ actions.js:133
(anonymous) @ index.js:8
(anonymous) @ ValidationStatus.js:58
l @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ ValidationStatus.js:57
s @ react-dom.production.min.js:14
h @ react-dom.production.min.js:14
(anonymous) @ react-dom.production.min.js:14
g @ react-dom.production.min.js:15
st @ react-dom.production.min.js:52
ot @ react-dom.production.min.js:51
ut @ react-dom.production.min.js:52
pt @ react-dom.production.min.js:56
D @ react-dom.production.min.js:287
z @ react-dom.production.min.js:19
Qt @ react-dom.production.min.js:70
Jt @ react-dom.production.min.js:69
t.unstable_runWithPriority @ scheduler.production.min.js:19
Gi @ react-dom.production.min.js:122
P @ react-dom.production.min.js:287
Zt @ react-dom.production.min.js:68
n @ helpers.js:72
Show 71 more frames
asyncToGenerator.js:6 Uncaught (in promise) TypeError: Failed to fetch
```

**Comment by koeppelmann** [comment]
@adzialocha for more context:
https://blockscout.com/poa/xdai/tx/0x8392780bc95ee2c1a412625268ae7d364c92082059bd8fc46e51b3d61915d489/internal-transactions

It seems like the Safe was successfully created but not the token contract?

**Comment by ligi** [comment]
After some more reloads I could now get a step further. Feel free to close the issue - maybe also just a consequence of the overloaded servers.

**Comment by tschubotz** [comment]
I'm experiencing the same issue. Endless spinner at "Your circles account is loading" and then it just resets to where I can click the button "create account".
Account is `0x7087C0338aA7Cd34Ba918A7fC14B39b18b7a85Ab`
Please let me know if there's any further info I could provide for debugging, happy to help.

**Comment by adzialocha** [comment]
Hi all! Thanks for the reports!

We're facing heavy infrastructure problems with a naive server setup and 70 million requests per 24 hours. The problems reported here are all related to this and we're working hard on improving it so the app runs more stable. With our team capacity and knowledge we are not prepared for such task and reaching out for help currently.

Undeployed Tokens are related to failed relayer tasks due to the outages, the process can be picked up again by:

1. Resetting your wallet (keep seed phrase)
2. Login via seed phrase
3. Try deploy again

That all "should" work given that the servers don't go down, but this currently we can't guarantee. Thank you for your understanding :sweat_smile:

**Comment by tschubotz** [comment]
I tried picking up the process with the 3 steps you suggested. Unfortunately it didn't work.

But no worries, will just wait for the proper fix! 

70M requests/day is quite a lot 😳

**Comment by axic** [comment]
```
validation:1 Access to fetch at 'https://relay.circles.garden/api/v2/safes/0xDa1523dB637fA18F9c03b854A34E31A7BE9D1Cd8/funded/' from origin 'https://circles.garden' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

@ligi has a CORS error with `relay.circles.garden`, but interestingly CORS is fine with that endpoint here, but fails with `api.circles.garden`:
```
Access to fetch at 'https://api.circles.garden/api/users/?address[]=0x...' from origin 'https://circles.garden' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

Browser is Brave.

**Comment by axic** [comment]
I also had a [different problem earlier today](https://twitter.com/alexberegszaszi/status/1318191265628946432):
![image](https://user-images.githubusercontent.com/20340/96495878-66fd1100-1240-11eb-8e7e-35eacdae4e1f.png)

This was fixed and now stuck in a different problem: `POST https://relay.circles.garden/api/v2/safes/0x...transactions/estimate/` fails with:
```
{"exception":"CannotEstimateGas: Received 0x for tx={'value': 0, 'chainId': 100, 'from': '...', 'gasPrice': 0, 'to': '...', 'data': '...'}"}
```

**Comment by axic** [comment]
It turned out my problems on Brave seem to be fixed with cleaning cookies (e.g. in running in a new private tab).

**Comment by zhous** [comment]
**First one(with Chrome) works, another one is still loading so I'll close it right now.**

I've been stuck for more than one week with Chrome. Not constantly working on it everyday.
After 3 trusts, This morning I tried to create my account but failed (I had no xDAI on my wallet). 
Then I used the Seed with Firefox. But it says the name is used. So I used another name and another Email address. It created a different profile for me.

Issues:
1. Now I have two profile address, which are all 3 trusted!
2. Succeeded to send some xDAI to the first wallet. 
[https://blockscout.com/poa/xdai/address/0xFBB9cb39fEFb9Ccc00AF94D47C43d03d5615141A/transactions](https://blockscout.com/poa/xdai/address/0xFBB9cb39fEFb9Ccc00AF94D47C43d03d5615141A/transactions)
But have failed several times while sending some xDAI to the later one!
[https://blockscout.com/poa/xdai/address/0x58edE2E89254e8FaECEAfBfE3BF5AeD4641d0cb7/transactions](https://blockscout.com/poa/xdai/address/0x58edE2E89254e8FaECEAfBfE3BF5AeD4641d0cb7/transactions)
3. First wallet with Chrome, second wallet with Firefox, there're always "Your Circles account is loading!" when trying to create my account. Finally Chrome got it through.
